ssr-express/README.md

```diff
@@ -0,0 +1,131 @@
+- Pages that do not start with `/amp/` will have the URL rewritten to the equivalent page by appending
+the prefix to URL and the output will be transformed. eg: `https://example.com/page.html` is rewritten
+to `https://example.com/amp/page.html`.
+
+Example configuration using the default configuration and a static middleware:
+```javascript
+const express = require('express');
+const path = require('path');
+const app = express();
+const createAmpSsrMiddleware = require('amp-toolbox-ssr-express');
+const ampSsr = require('amp-toolbox-ssr');
+
+ampSsr.setConfig({
+  transformers: [
+    'AddAmpLink',
+    'ServerSideRendering',
+    'RemoveAmpAttribute',
+    // needs to run after ServerSideRendering
+    'AmpBoilerplateTransformer',
+    // needs to run after ServerSideRendering
+    'ReorderHeadTransformer'
+  ]
+});
+
+// Setup the AMP-SSR Transformer and pass along the path to build the link tag.
+const ampSsrMiddleware = createAmpSsrMiddleware(ampSsr);
+
+// It's important that the transformMiddleware is added BEFORE the static middleware.
+// This allows us to replace the parts needed before static handles the request.
+app.use(ampSsrMiddleware);
+
+const staticMiddleware = express.static(path.join(__dirname, '/public'));
+app.use(staticMiddleware);
+```
+
+### Using a custom configuration
+For developers who want to use a different URL scheme than prefixing the AMP files with
+`/amp`, it is possible to add a configuration with custom methods to decide if a page
+should be transformed and to transform an URL to the AMP equivalent.
+
+The example bellow transforms uses the `?amp` query parameter to indicate pages that are
+AMPs. eg: `https://example.com/page.html` becomes `https://example.com/page.html?amp`.
+
+```javascript
+// skipTransform receives a `string` containing an URL as parameter, and returns a `boolean`.
+const skipTransform = url => {
+  // http://example.com is used as the second parameter,
+  // as the URL constructor requires a valid domain.
+  const parsedUrl = new URL(url, 'https://example.com');
+  return parsedUrl.searchParams.has('amp');
+};
+
+// ampUrl receives a `string` containing an URL as parameter, and returns a `string` indicating
+// the equivalent AMP URL.
+const getAmpUrl = url => {
+  // http://example.com is used as the second parameter,
+  // as the URL constructor requires a valid domain.
+  const parsedUrl = new URL(url, 'https://example.com');
+  parsedUrl.searchParams.set('amp', '');
+  return parsedUrl.pathname + parsedUrl.search;
+};
+
+// Setup the AMP-SSR Transformer and pass along the path to build the link tag.
+const ampSsrMiddleware = createAmpSsrMiddleware(ampSsr, {
+    skipTransform: skipTransform,
+    getAmpUrl: getAmpUrl
+});
+
+// It's important that the transformMiddleware is added BEFORE the static middleware.
+// This allows us to replace the parts needed before static handles the request.
+app.use(transformMiddleware);
+```
+## Cache server-side-rendered AMPs
```

Comment:
Maybe call this out as a best practice/optional optimization: "Best Practice: Cache server-side-rendered AMPs".

Reply:
I'll rename to Best Practice.